src/packages/BalancedJs/contracts/Baln.ts

```diff
@@ -56,4 +56,29 @@ export default class Baln extends Contract {
     console.log(payload);
     return this.callIconex(payload);
   }
+
+  public async transfer(toAddress: string, value: number): Promise<any> {
+    const callParams = this.transactionParamsBuilder({
+      method: 'transfer',
+      params: {
+        _to: toAddress,
+        _value: '0x' + IconAmount.of(value, IconAmount.Unit.ICX).toLoop().toString(16),
+      },
+      value: 0,
+    });
+
+    return this.callIconex(callParams);
+  }
+
+  async stakeBALN(value: number): Promise<ResponseJsonRPCPayload> {
```

Comment:
I'd like to use the names of the same method as the names of the smart contract method.

Reply:
stakedBALN -> stake